lablib/lib/utils.py

```diff
@@ -14,19 +14,59 @@
 log.setLevel(logging.DEBUG)
 
 
-def __call_cmd(cmd: List[str], timeout=0, retries=0) -> Optional[str]:
+def __get_lablib_env() -> None:
+    _parts = Path(__file__).parts[:-3]
+    vendor_root = Path(*_parts, "vendor")
+    env = os.environ.copy()
+
+    if oiio_root := env.get("LABLIB_OIIO"):
+        log.debug(f"Using oiiotool from {oiio_root}")
+    else:
+        log.warning("LABLIB_OIIO environment variable not set. Using default.")
+        oiio_root = Path(vendor_root, "oiio", "windows")
+
+    if ffmpeg_root := env.get("LABLIB_FFMPEG"):
+        log.debug(f"Using ffmpeg from {ffmpeg_root}")
+    else:
+        log.warning("LABLIB_FFMPEG environment variable not set. Using default.")
```

Comment:
Lets turn this and others into log.info rather then warning. 
```suggestion
        log.info("LABLIB_FFMPEG environment variable not set. Using default.")
```

Reply:
adressed in 9203c92e64755ad97d8221c1e5f84fcf99729881